docs/developer-documentation.md

```diff
@@ -35,6 +35,19 @@ _(Most of the following information pertains to the Production version (Uzbl-bas
 All of the files/folders from the Github repo should be cloned into this directory.
 ```
 
+```
+/home/pi/screenly/bin/
+
+build_containers.sh -> script to build docker cross-compiled images and different architecture containers
```

Comment:
Can you turn this in to bullet points instead?

Reply:
yup